.stylelintrc

```diff
@@ -24,7 +24,8 @@
     "declaration-colon-space-before": "never",
     "no-descending-specificity": null,
     "order/properties-alphabetical-order": true,
-    "length-zero-no-unit": true
+    "length-zero-no-unit": true,
+    "selector-pseudo-element-no-unknown": [true, { "ignorePseudoElements": ["ng-deep"] }]
```

Comment:
We should not allow the use of "ng-deep" in all the places. Instead, use a pragma/disable-flag in the file where it's needed with an explanation of why it's needed.

Reply:
Done

Reply:
@Patel-Muhammad Just to recheck did you check the frontend working after making this change?